docs/manual/troubleshooting.tex

```diff
@@ -1073,6 +1073,31 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
 \end{enumerate}
 
 
+\subsection{Configure Eclipse to edit the Checker Framework\label{building-eclipse}}
+
+\begin{enumerate}
+\item Follow the instructions earlier in Section~\ref{build-source} to
+clone and build all projects from their sources.
+\item Download Eclipse from
+the \href{https://www.eclipse.org/downloads}{official Eclipse website}
+and install it.
+\item Run Eclipse and set the
+workspace to the directory pointed to by \code{\$JSR308}.
+\item Enter the main Eclipse working screen and in the ``File'' menu,
+select ``Import'' $\rightarrow$ ``General'' $\rightarrow$ ``Existing
+Projects into workspace''.
+\item After the ``Import Projects'' window appears, select ``Select
+Root Directory'', and browse for \code{\$JSR308} folder.
```

Comment:
`browse for \code{\$JSR308} folder.` ->
`select the directory pointed to by \code{\$JSR308}.`

Reply:
@Sadaf91 Can you make this slight wording change?